content/_data/changelogs/lts.yml

```diff
@@ -3913,6 +3913,590 @@
+
+  # Major user experience improvements
+  - type: major rfe
+    category: major rfe
+    pull: 4752
+    authors:
+    - 95jonpet
+    - timja
+    - fqueiruga
+    references:
+      - issue: 60924
+      - pull: 4752
+      - issue: 62515
+      - pull: 4763
+      - pull: 4772
+      - pull: 4814
+      - pull: 4842
+      - pull: 4843
+      - url: https://github.com/jenkinsci/dark-theme
+        title: Dark Theme repository
+      - url: https://cd.foundation/blog/2020/08/04/introducing-the-jenkins-dark-theme/
+        title: Introducing the Jenkins Dark Theme
+    message: |-
+      Release 'alpha' dark theme.
+
+  # Major Windows Support Updates
+  - type: major rfe
+    category: major rfe
+    pull: 4823
+    references:
+      - url: https://www.jenkins.io/blog/2020/07/23/windows-support-updates/
+        title: announcement
+      - url: https://www.jenkins.io/blog/2020/07/23/windows-support-updates/#use-cases-affected-by-net-framework-2-0-support-removal
+        title: upgrade guidelines
+      - issue: 60005
+      - issue: 61862
+      - url: https://www.jenkins.io/doc/administration/requirements/windows/
+        title: Windows support policy
+    authors:
+    - NextTurn
+    - oleg-nenashev
+    message: |-
+      Stop supporting .NET Framework 2.0 for launching Jenkins server and agents as a Windows service.
+      .NET Framework 4.0 or above is now required.
+  - type: major rfe
+    category: major rfe
+    pull: 4823
+    references:
+      - url: https://www.jenkins.io/blog/2020/07/23/windows-support-updates/#windows-service-management-changes-in-jenkins-2-248
+        title: changes summary
+      - url: https://github.com/winsw/winsw/releases
+        title: full WinSW changelog
+      - url: https://github.com/jenkinsci/windows-slave-installer-module/releases/tag/windows-slave-installer-2.0
+        title: Windows Agent Installer 2.0 changelog
+    authors:
+    - NextTurn
+    - oleg-nenashev
+    message: |-
+      Update Windows Service Wrapper (WinSW) from 2.3.0 executable for .NET Framework 2.0 to 2.9.0 for .NET Framework 4.0.
+      Includes numerous improvements and bugfixes.
+      Most notably, the service installer will now ask for permission elevation if the required.
+
+  # Plugin manager improvements
+  - type: rfe
+    category: rfe
+    pull: 4858
+    authors:
+    - fqueiruga
+    message: |-
+      Align the Plugin Manager table headers.
+  - type: rfe
+    category: rfe
+    pull: 4742
+    issue: 62332
+    authors:
+    - daniel-beck
+    message: |-
+      Show in plugin manager when newer releases of plugins exist but aren't being offered due to unsatisfied requirements.
+  - type: rfe
+    category: rfe
+    pull: 4073
+    issue: 59136
+    authors:
+    - jetersen
+    - daniel-beck
+    message: |-
+      Show warnings for <b>deprecated</b> plugins in the update manager and administrative monitors.
+
+  # More user experience improvements
```

Comment:
Combine PRs 4808 4782 4700 4816 4820 4835 into one generic "Various improvements to give the Jenkins UI a more modern look and feel" entry.